### PR TITLE
Prepublish script for es5

### PR DESCRIPTION
**Pull request by tonypizzicato**
See #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
   "author": "OpenZipkin <openzipkin.alt@gmail.com>",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
```
Comment (eirslett) [inline package.json]: I think we should be able to get away with only using babel-preset-env and then target node 4.

Comment (eirslett) [inline package.json]: Another option would be to just run the whole code base through babel to make it node 4-compatible, and then just set up the build process etc. to use node 4. So we don't need a build step.

Comment (tonypizzicato) [inline package.json]: ok, makes sense. 
but pr was opened for browser specific packages support and running transpiling step before publishing to npm

Comment (eirslett) [inline package.json]: Yes, using it from browsers is a good point... in that case it's probably best to do it like you have, use babel-preset-es2015 and let that be used from both browsers and node servers.

Comment (tonypizzicato) [inline package.json]: so, what do you think about merging this pr and publishing new version to npm?

Comment (eirslett) [inline package.json]: LGTM! @adriancole?

**Comment by adriancole** [comment]
maybe add a description to the PR and fix the type in the commit message? :) I added the issue link.

testing locally now. thanks for the help!

**Comment by adriancole** [comment]
also cc @openzipkin/devops-tooling

**Comment by adriancole** [comment]
so nothing broke for me, though my ability to test this is limited. Does this need any README changes, yet? Ex the readme talks about node 6

**Comment by eirslett** [comment]
Looks like the build is failing because eslint is linting the generated code. We only want to lint the original code.

**Comment by sveisvei** [comment]
Btw, this PR should use `git mv` so we don`t loose history.

**Comment by tonypizzicato** [comment]
@sveisvei update pr with two commits: one with `git mv` that is treated as file renaming. but after changes in the second pr(fixing `require` paths) it's treaded as added/deleted files again compared to master

**Comment by adriancole** [comment]
thanks for the update. ps lint config still needs an update as the build is broke

**Comment by adriancole** [comment]
this change is still pending lint

**Comment by tonypizzicato** [comment]
@adriancole updated .eslintignore

**Comment by adriancole** [comment]
thanks for the work and follow-up @tonypizzicato

**Pull request merged**